helpers/lib/cdnify.js

```diff
@@ -16,38 +16,45 @@ module.exports = globals => {
         const editSessionId = siteSettings.theme_session_id;
         const cdnSettings = themeSettings.cdn;
 
-        const protocolMatch = /(.*!?:)/;
-
         if (path instanceof globals.handlebars.SafeString) {
             path = path.string;
         }
 
         if (!path) {
             return '';
         }
-
-        if (/^(?:https?:)?\/\//.test(path)) {
+        
+        if (path.startsWith('//') || path.startsWith('http')) {
             return path;
         }
 
         if (['.', '/'].includes(path[0])) {
-            path = path.replace(/^[\.\/]+/, '');
+            for (var i = 0; i < path.length; i++) {
+                if (path[i] !== '.' && path[i] !== '/') {
```

Comment:
This doesn't match current behavior. This block is meant to remove leading dots and slashes from the beginning of the string.

Reply:
That is what the for loop is doing.

Reply:
Ok, I see now. Maybe adding some comments would help.

Reply:
fixed.